tests/integration/test_charm.py

```diff
@@ -4,6 +4,8 @@
 
 """Integration tests for Flask charm."""
 
+# pylint: disable=too-many-arguments
```

Comment:
Should this include an explanatory comment why this disable is needed?

Reply:
Yes, it's caused by pytest fixtures. I have added that comment.